examples/burn-cpu.yaml

```diff
@@ -8,12 +8,10 @@ spec:
   selector:
     labelSelectors:
       "app.kubernetes.io/component": "tikv"
-  stressors: 
-    cpu: 
+  stressors:
+    cpu:
       workers: 1
       load: 100
       options: ["--cpu 2", "--timeout 600", "--hdd 1"]
   duration: "30s"
-  scheduler:
```

Comment:
So we would not test scheduler seperately in every chaos. Instead we could test it once and for all?

Reply:
Yes. But we nearly didn't test scheduler in previous tests :facepalm: .

I tried to write an integration test for the scheduler, through the `sigs.k8s.io/controller-runtime/pkg/envtest`. Would you like to help and make it easy to define and add case ? (And I think it could be added in another PR, as this one is big enough :facepalm: )

Reply:
Sure. I'll check it out.

Reply:
umm, I'm a bit coinfused. Does that mean we would use `envtest` to create a virtual cluster, register crd and controllers, and run tests on it?

Reply:
Yes. Some tests are already using it, e.g. `api/v1alpha1/suite_test.go`.

Reply:
It seems that `suite_test.go` is good to use on crds, but it has no controllers. Could we just reuse that?